package.json

```diff
@@ -28,6 +28,9 @@
   "dependencies": {
     "matched": "^0.4.4"
   },
+  "peerDependencies": {
+    "rollup": ">=0.48.0"
```

Comment:
I changed the peerdep to include all future releases in its range. Since rollup releases so often, this may be less painful. It will just break in the future as it did this time.